packages/dev/core/src/Debug/physicsViewer.ts

```diff
@@ -122,6 +130,36 @@ export class PhysicsViewer {
         return debugMesh;
     }
 
+    /**
+     *
+     */
+    public showBody(body: PhysicsBody): Nullable<AbstractMesh> {

```

Comment:
will that fail when using physics v1?

Reply:
It will fail before you can call this method. Physics Plugin version is checked in the PhysicsBody/Shape/Material constructors.